rpc/core/env.go

```diff
@@ -30,7 +30,7 @@ const (
 
 	// genesisChunkSize is the maximum size, in bytes, of each
 	// chunk in the genesis structure for the chunked API.
-	genesisChunkSize = 16 * 1024 * 1024 // 16
+	genesisChunkSize = 2 * 1024 * 1024 // 2 MB
```

Comment:
Based on the fact that too frequent requests might be considered as DOS attacks, not sure if it is the best idea to lower this this much.

Reply:
that's a good point, we don't have a rate limiter to the RPC requests (only in P2P) in my understanding is that the operators implement their own rate limiter via proxy servers or load balancers. I think this parameter should be part of the config since it should be dynamic (respect or be in sync with external rate-limiters)

Reply:
@jmalicevic that's a valid concern, but I agree with @andynog that it's the operator's responsibility to have rate limiting (e.g., via reverse-proxies or load balancers) in place.

Sure, a higher chunk size means less RPC requests to get the whole genesis, but I think that nothing prevents a malicious client from trying to DDoS a node by spamming requests to `/genesis_chunked` anyway. They can do so regardless of the chunk size.

Reply:
Ok, let's reduce it but please add a changelog entry as this might break people having scripts to read the chunk sizes in place. So both as a fix and under breaking changes.